### DISCUSSION TITLE
本科一本院校，电子信息类，已毕业一年，绩点2.96不算高，家里不给经济支持，如何低成本润去欧美？

### DISCUSSION THREAD

**Discussion by ghost** [The-Run-Philosophy-Organization/run#593] [Q&A]
绩点这个真没办法，都毕业了。当时在课堂上确实学不会，中间还休学了一年。
报过培训班，后来也自己找网盘上的盗版课，对渗透测试和嵌入式很感兴趣。
英语四级474分，六级没考过，英语不算太好。
高考在高考大省，45万理科生里排名两万多名。
个人想法是在国内赶紧学好一门技术，在北京找个技术工作攒够钱，出国读计算机硕士。
二十万rmb攒个三年有可能攒到。
本人是女生。
求移民成功的前辈指点。
本专业绩点都不高，休学前在班里处于25/55水平，但是挂科我当然愁，大三开学一个月就知道好几门课这学期会挂，又有其他的压力，就抑郁了。

**Comment by ethenlong** [comment]
刷雅思，去加拿大读研文，两年毕业，有三年开放工签，够留下来了。
